Sources/VLCMovieViewController.m

```diff
@@ -1234,39 +1295,65 @@ - (IBAction)sleepTimerAction:(id)sender
 
 - (void)moreActions:(UIButton *)sender
 {
-    if (_multiSelectionView.hidden == NO) {
-        [UIView animateWithDuration:.3
-                         animations:^{
-                             self->_multiSelectionView.hidden = YES;
-                         }
-                         completion:^(BOOL finished){
-                         }];
-        return;
-    }
+    #if !NEW_UI
+        if (_multiSelectionView.hidden == NO) {
+            [UIView animateWithDuration:.3
+                             animations:^{
+                                 self->_multiSelectionView.hidden = YES;
+                             }
+                             completion:^(BOOL finished){
+                             }];
+            return;
+        }
 
-    if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone) {
-        if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])) {
-            _multiSelectionView.showsEqualizer = YES;
-        } else {
-            _multiSelectionView.showsEqualizer = NO;
+        if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone) {
+            if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])) {
+                _multiSelectionView.showsEqualizer = YES;
+            } else {
+                _multiSelectionView.showsEqualizer = NO;
+            }
```

Comment:
in general we would want to write but I know you didn 't write and touch it so just for future references 
Bool isInLandscape = UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])
_multiSelectionView.showsEqualizer = isInLandscape